docs/pages/actions/apollo/api-reference/create-call.mdx

```diff
@@ -11,10 +11,15 @@ curl --request POST \
   --header 'x-vessel-api-token: ' \
   --header 'x-vessel-access-token: '\
   --data '{
-  "userId": "123123",
   "contactId": "456456",
   "dispositionId": "789789",
-  "note": "example note"
+  "purposeId": "789789",
+  "note": "example note",
+  "fromNumber": "555-555-5555",
+  "toNumber": "555-555-5555",
```

Comment:
Is this The formatting Apollo expects the phone number to be in?

Reply:
I tested it with those values, it seemed to work. Though I wasn't able to get the UI to show it specifically

Reply:
Like the response back had the same phone number format I provided it